docs/guides/main.rst

```diff
@@ -9,12 +9,57 @@ collection better.
 Installing
 ----------
 
-You will need Python. Beets works on Python 3.8 or later.
+Beets requires Python 3.9 or later, you will need to install that first.
+Depending on your operating system, you may also be able to install beets from a
+package manager, or you can install it with pipx_ or pip_.
+
+Using pip(x)
+~~~~~~~~~~~~
+
+To use the most recent version of beets, we recommend installing it with pipx_,
+the Python package manager. If you don't have pipx_ installed, you can follow
+the instructions on the `pipx installation page`_ to get it set up.
+
+.. code-block:: console
+
+    pipx install beets
+
+If you prefer to use pip_, you can install beets with the following command:
+
+.. code-block:: console
+
+    pip install beets
+    # or, to install for the current user only:
+    pip install --user beets
+
+.. attention::
+
+    Python 3.13 is not officially supported yet!
+
+    If you are using Python 3.13, please be aware that it is not officially
+    supported yet. You may encounter issues, and we recommend using Python 3.12
+    or earlier until support is confirmed.
+
+.. _pip: https://pip.pypa.io/en/
+
+.. _pipx: https://pipx.pypa.io/stable
+
+.. _pipx installation page: https://pipx.pypa.io/stable/installation/
+
+Using a Package Manager
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on your operating system, you may be able to install beets using a
+package manager. Here are some common options:
+
+.. attention::
+
+    Package manager installations may not provide the latest version of beets.
+
+    Release cycles for package managers vary, and they may not always have the
+    most recent version of beets. If you want the latest features and fixes,
+    consider using pipx_ or pip_ as described above.
```

Comment:
I think it would be good to also mention that system package managers make it much harder to install plugins